pkg/api/v1/schema.go

```diff
@@ -810,6 +810,10 @@ type DHCPOptions struct {
 	// If specified will pass option 66 to interface's DHCP server
 	// +optional
 	TFTPServerName string `json:"tftpServerName,omitempty"`
+
+	// If specified will pass extra DHCP options for private use, range: 224-254
```

Comment:
@davidvossel  added more description on this new option and have marked it //optional. thank you :)